repository-data/application/src/main/resources/hcm-config/namespaces/publicationsystem/series.yaml

```diff
@@ -55,9 +55,26 @@ definitions:
             wicket.css:
             - document-summary
             wicket.id: ${cluster.id}.left.item
+          /ShowLatest:
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+            caption: Show First Publication as Latest
```

Comment:
Just wondering about wording for the end user.  Does "Show **First** Publication as Latest" mean "Show **most recent** publication as Latest"?  Should this field be ticked by default?

Reply:
`Most recent` would be better 👍 
Left off by default as there are quite a few more that we want off than on.